app/src/main/java/org/gdg/frisbee/android/chapter/NewsFragment.java

```diff
@@ -198,24 +194,9 @@ public void onPostExecute(String[] params, ActivityFeed activityFeed) {
     }
 
     @Nullable
-    private ActivityFeed getActivityFeedSync(String param) {
-        try {
-            Plus client = App.getInstance().getPlusClient();
-
-            Plus.Activities.List request = client.activities().list(param, "public");
-            request.setMaxResults(10L);
-            request.setFields(
-                "nextPageToken,"
-                    + "items(id,published,url,object/content,verb,"
-                    + "object/attachments,annotation,object(plusoners,replies,resharers))");
```

Comment:
I think we lost `setMaxResults` here. I wonder what is the default value. The default value can be better for us